app/views/pages/rollover.html.erb

```diff
@@ -37,7 +37,7 @@
     <ul class="govuk-list govuk-list--bullet govuk-!-margin-bottom-4">
       <li>degree requirements</li>
       <li>GCSE requirements</li>
-      <li>whether you can sponsor visas</li>
+      <li>whether visas can be sponsored</li>
```

Comment:
Where would this show?

Reply:
on the rollover interstitial page (seen when first logging in during rollover I think)

Reply:
This is what the rollover interstitial says at the moment:

![Screenshot 2021-09-02 at 11 37 16](https://user-images.githubusercontent.com/85290597/131829403-efddece8-9e2c-49ae-bf39-6472b0d10e05.png)

Reply:
Yep - should we change the 3rd bullet?

Reply:
Yep 'if candidates can get visa sponsorship'

Reply:
Done. Looks like we didn't change it to match the prototype before, it was still saying "whether..." - doh!